### ISSUE TITLE
Transfer childrenhelpingscience.com domain

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1190]
**TL;DR**: Move childrenhelpingscience.com into our possession and then have it point to our current production deployment.

**Narrative**
These are the instructions on how to complete the domain transfer and set up the domain.  

**Implementation Notes**

Log into Google Domains and get to the website where we can enter in the transfer code needed to move the domain.  

1. Log into [Google Domains](https://domains.google.com).
2. Click "Transfer" in the left menu.
3. Enter `childrenhelpingscience.com` into the field at the top of the page and press `enter`. <img width="849" alt="Screenshot 2023-04-25 at 10 11 57 AM" src="https://user-images.githubusercontent.com/44074998/234304177-cbbf6b2c-16a3-45b5-a1e5-757d6acc7759.png">
4.  Once you get to the view asking for the registration code leave that tab open.  In the next step we'll get the code from squarespace. <img width="848" alt="Screenshot 2023-04-25 at 10 13 30 AM" src="https://user-images.githubusercontent.com/44074998/234304693-edb52860-16b7-4685-bd27-85321edd8b74.png">

Next, we'll need to get a transfer code from the current registrar, which is squarespace for our case. 

1. Get access to the childrenhelpingscience.com squarespace account.  This would require contacting Mark Sheskin. 
1. Go [here](https://support.squarespace.com/hc/en-us/articles/205812338-Transferring-a-domain-away-from-Squarespace) to get detailed instructions on how to get the code.  
1. Once the code is acquired, enter it into the google domain transfer form from above.  

Import web settings. From google docs, this should be an auto import. I am suspecting that this will import the childrenhelpingscience domain information and will have to be replaced with lookit.mit.edu domain information.  

**Todo** Get the correct domain information from the test run.  

Configure registration settings.  Again from google's docs, it is recommended to find the "[Privacy protection](https://support.google.com/domains/answer/3251242?hl=en)" and "[domain Auto-renew](https://support.google.com/domains/answer/6093180?hl=en)" value and set them to our preferences.  

Check out.  I am assuming this will require a credit card, so we might need to have one handy.

**Comment by mekline** [comment]
We have a credit card to use for this  - so maybe what would make sense is for me to purchase the test domain, and to then initiate this transfer - does that make sense?

**Comment by okaycj** [comment]
This has been completed.
